### ISSUE TITLE
[Broken Website] Bing.com - Bing chat never loads with Dark mode enabled

### ISSUE THREAD

**Issue by Morac2** [darkreader/darkreader#12086]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Using the iOS version of the Dark Mode app on my iPad 11 running iPadOS 17.2, if I search for anything on Bing.com with Dark Reader’s dark mode enabled on the site, the bing Chat box never loads.  I have to turn off Dark mode on the site to get Bing Chat to load. 

### Website Address

https://www.bing.com/

### Steps To Reproduce

1. Search for anything on Bing in the Ask me anything box with dark enabled.
2. The Bing Chat box never loads and shows a blue line going back and forth. 

### Screenshots

<img width="447" alt="image" src="https://github.com/darkreader/darkreader/assets/1596194/d8c38bba-34dc-4062-92e7-5e5ef4b7547a">

### Operating System

iPadOS 17.2

### Web Browser name and version

Mobile Safari 17.2

### Dark Reader version

1.4.4

### Additional Context

Disabling Dark Reader on Bing.com is a work around.

**Comment by MSBlueVn** [comment]
That's the sidebar, the full chat work, but slowly, as reported here:
https://github.com/darkreader/darkreader/issues/12122

**Comment by Morac2** [comment]
It doesn’t output at all.  The sidebar never loads with DarkReader enabled.

**Comment by MSBlueVn** [comment]
Yeah, the sidebar is annoying, but I use the full screen chat, that works but slowly.
